test/error_page_test.bats

```diff
@@ -10,6 +10,7 @@ setup() {
 }
 
 @test "returns custom error page" {
+  etcdctl set /kontena/haproxy/lb/services/service-b/virtual_hosts www.foo.com
```

Comment:
Why is this needed?

Reply:
hmm, that also means that if I'd configure `KONTENA_LB_HEALTH_URI` to create LB level "healthcheck" it woulnd't work either?

Reply:
Yes.

Reply:
If user has only tcp-services, health checks should be done at tcp (port) level. If user has http-services then we should not open http port until we have proper config from etcd.

Reply:
Fair enough, just wanted to make sure this is a known design choice. :)